### ISSUE TITLE
Umbraco 8 gridContent is null

### ISSUE THREAD

**Issue by joepvtl** [umbraco/Umbraco-CMS#7058]
After migration from v7 to 8  I try to open a node that has nestedcontent that contains a grid but it gives a nullref exception. So I took a look at what happens inside of umbraco.
![image](https://user-images.githubusercontent.com/42842226/68212690-55443e80-ffda-11e9-861d-a4f704b02aed.png)

As you can see the val variable is a empty string. I don't why this is the case because when I look inside of the database is says that "gridContent : null".

So once it continues it gives the nullref here because when it deserializes the empty string the grid variable gets set to null.

![image](https://user-images.githubusercontent.com/42842226/68212831-a3594200-ffda-11e9-8401-4ed074522fb2.png)

I deleted the "gridContent : null" from the json because I thought this had something to do with the migration. After I did this could open it again but when I saved the node the error came back.

**Comment by stevemegson** [comment]
It looks like this line in `NestedContentPropertyValueEditor.ToEditor` is the source of the empty strings where we expect nulls:

```c#
tempProp.SetValue(propValues[propAlias] == null ? null : propValues[propAlias].ToString());
````

When the JSON contains `null`, `propValues[propAlias]` isn't null. It's a `JValue` with its `Type` property set to `Null`, and calling `ToString()` returns an empty string.

**Comment by nul800sebastiaan** [comment]
I am actually not sure if we support putting a grid inside a nested content item. 

But if we do need to update this `ToEditor` method that would introduce a breaking change since the behavior is changing and people are possibly relying on the previous behavior.

**Comment by joepvtl** [comment]
`if (string.IsNullOrEmpty(val as string)) return string.Empty;` I don't think that this is a breaking change right?

**Comment by kjac** [comment]
Fixed by #7080
